packages/dialect-react-sdk/context/DialectContext/Wallet/index.ts

```diff
@@ -15,10 +16,38 @@ export interface DialectWalletStatesHolderState {
   isEncryptingState: State<boolean>;
 }
 
+export interface HardwareWalletConfig {
+  hardwareWalletEnabled: boolean;
+}
+
 function useDialectWalletStatesHolder(): DialectWalletStatesHolderState {
   const [walletConnected, setWalletConnected] = useState(false);
   const [connectionInitiated, setConnectionInitiated] = useState(false);
-  const [hardwareWalletForced, setHardwareWalletForced] = useState(false);
+
+  const [
+    localStorageHardwareWalletConfig,
+    setLocalStorageHardwareWalletConfig,
+  ] = useLocalStorage<HardwareWalletConfig>('dialect-wallet-config', {
+    hardwareWalletEnabled: false,
+  });
```

Comment:
let's extract initialValue into const outside of component?

Reply:
Fixed.